class UserNotFoundException(Exception):
    pass


class UserAlreadyRegisteredException(Exception):
    pass
